src/modules/storage/BlockStorage.ts

```diff
@@ -74,30 +76,46 @@ export class BlockStorage extends Storages
                 (data.header.validators == undefined) ||
                 (data.header.validators._storage == undefined) ||
                 (data.header.merkle_root == undefined) ||
-                (data.header.signature == undefined)
+                (data.header.signature == undefined) ||
+                (data.txs == undefined)
         ) {
             if (callback != undefined)
                 callback("Parameter validation failed.");
             return;
         }
 
+        var tx_count: number = 0;
+        var enrollment_count: number = 0;
+
+        tx_count = data.txs.length;
+
+        console.log(tx_count);
```

Comment:
Random `console.log` aren't great. Either remove or add an explanation message. In this case, I would say remove.

Reply:
Removed it. This is a remnant of Debug.